e2e-tests/functions

```diff
@@ -715,6 +716,19 @@ deploy_cert_manager() {
 	sleep 30
 }
 
+delete_crd() {
+	kubectl_bin delete psmdb-backup --all || :
+	kubectl_bin delete psmdb-restore --all || :
+	kubectl_bin delete psmdb --all || :
+
+	kubectl_bin get crd/perconaservermongodbs.psmdb.percona.com \
+		&& timeout 120 kubectl delete -f "${src_dir}/deploy/crd.yaml" || true
+
+	kubectl_bin get crd/perconaservermongodbs.psmdb.percona.com \
+		&& timeout 120 kubectl delete -f "${src_dir}/deploy/crd.yaml"
```

Comment:
if the deletion above succeeds, this command and the test will fail right?

Reply:
no, please see `kubectl_bin`  fun . We will have 'return 1' but not exit code >= 1